Add new guide for incident testing (#34539)
DataDog/documentation: content/en/incident_response/incident_management/guides/test_incidents.md

---
title: Using Test Incidents
description: "Learn how to use test incidents to safely train your team, test automations, and validate your incident response process without affecting production data or triggering real notifications."
further_reading:
- link: "/incident_response/incident_management/setup_and_configuration/information#test-incidents"
  tag: "Documentation"
  text: "Configure test incident settings"
- link: "/incident_response/incident_management/setup_and_configuration/automations#testing-automations"
  tag: "Documentation"
  text: "Testing incident automations"
- link: "/incident_response/incident_management/investigate/declare"
  tag: "Documentation"
  text: "Declare an incident"
---

## Overview

{{< img src="/incident_response/incident_management/guides/test_incidents/example_test_incident.png" alt="Example of a test incident with a purple banner" style="width:100%;" >}}

> Test incidents are a type of incident designed for training, testing, and validation purposes. When enabled, test incidents allow your team to practice incident response workflows, validate automations, and familiarize new team members with your incident management process—all without impacting production analytics or triggering real notifications.

Reviewer: Tightening up slightly.

```suggestion
Test incidents are incidents used for training, testing, and validation. They allow your team to practice incident response workflows, validate automations, and onboard new team members without impacting production analytics or triggering real notifications.
```

> Test incidents are visually distinguished by a **purple banner** and are isolated from your production incident data.

Reviewer: ```suggestion
Test incidents display a **purple banner** and are isolated from production incident data.
```

## Default behavior

By default, test incidents:
- Do not appear in search results
- Do not appear in analytics
- Do not execute integrations
- Do not execute notification rules

During declaration, you have the option to enable any of these behaviors if needed for your testing purposes. For example, enable "Execute integrations" when testing workflows, or enable "Include in analytics" when validating reporting.

## Use cases

**Training and response drills**: Use test incidents to onboard responders and simulate realistic scenarios. Practice the full incident lifecycle, test escalation and on-call responsiveness, and evaluate team coordination, without impacting team metrics.

> **Testing incident automations and notifications rules**: Use test incidents to verify automations and notification rules trigger correctly, execute expected actions (for example, Slack messages or task creation), and can be refined without affecting production workflows.

Reviewer: Adding "that" to improve the flow -- my brain wants to stop at "verify automations and notification rules"; the "that" helps me anticipate that there's more coming.

```suggestion
**Testing incident automations and notifications rules**: Use test incidents to verify that automations and notification rules trigger correctly, execute expected actions (such as sending Slack messages or creating tasks), and can be refined without affecting production workflows.
```

**Validating integrations**: Create test incidents to confirm integrations (such as Jira, ServiceNow, or Slack) create and sync records correctly, propagate updates, and format notifications as expected.

## Enabling test incidents

> <div class="alert alert-danger">Only users with the appropriate Datadog administrative permissions can enable test incidents. After it's enabled, any user with the <strong>Incidents Write</strong> permission can declare test incidents</div>

Reviewer: I'm assuming "it's" refers to test incidents? Or are you talking about the permission? Either way, worth clarifying to avoid confusion!

```suggestion
<div class="alert alert-danger">Only users with the appropriate Datadog administrative permissions can enable test incidents. After test incidents are enabled, any user with the <strong>Incidents Write</strong> permission can declare them.</div>
```

> Test incidents must be enabled before use. To enable them:

Reviewer: ```suggestion
Test incidents must be enabled before use. To enable:
```

1. Navigate to [**Incidents** > **Settings** > **Information**][3]
2. Scroll to the **Test incidents** section
3. Toggle the setting to **Enabled** for your incident type

> 4. Click **Save Changes**

Reviewer: The "for your incident type" is throwing me off a bit. Can we remove that phrase? It makes it seem like there are multiple toggles to choose from but I only see one.
